tests/integration_tests/test_matchmaker_vetoes.py

```diff
@@ -10,6 +10,23 @@
 )
 
 
+async def test_used_pools_have_some_maps_and_all_map_names_are_unique(ladder_service):
+    used_map_pools = {}
+    for queue in ladder_service.queues.values():
+        for mq_map_pool in queue.map_pools.values():
+            pool_id = mq_map_pool.map_pool.id
+            if pool_id in [1, 2, 3]:
```

Comment:
on your first attempt there were 4 pool ids, and tests failed. so you reduced their amount to 3. why? wasn't the failure legitimate?